src/ert/config/model_config.py

```diff
@@ -37,6 +48,9 @@ def str_to_datetime(date_str: str) -> datetime:
 DEFAULT_JOBNAME_FORMAT = "<CONFIG_FILE>-<IENS>"
 DEFAULT_ECLBASE_FORMAT = "ECLBASE<IENS>"
 
+FULL_DISK_PERCENTAGE_THRESHOLD = 0.97
+MINIMUM_BYTES_LEFT_ON_DISK = 200 * 1024**3  # 200 GB required
```

Comment:
Hmm, we will get some complaints from all the Raspberry Pi users out there with a 16Gb memory card.

Reply:
Could we in some way leave this number for `ert-configurations` to set?

Reply:
The `0.97` is easier to accept as a general rule in Erts open source code.

Reply:
Should there be a comment in the source code on this?